homeassistant/components/hue/light.py

```diff
@@ -463,9 +404,7 @@ def device_info(self):
         else:
             await self.bridge.async_request_call(self.light.set_state(**command))
 
-    async def async_update(self):
```

Comment:
The downside with removing this is that we don't allow manual updates via the update service.

Reply:
I'll add it back. It won't hurt as we have should_poll=False, so it won't be polled.